CHANGELOG.md

```diff
@@ -175,6 +175,9 @@ Add Brazilian Portuguese translation (pt-BR.json) by [@pstwh](http://github.com/
 
 <img width="951" alt="image" src="https://user-images.githubusercontent.com/1778297/206615305-4c52031e-3f7d-4df2-8805-a79894206911.png">
 
+### Add Waveform Visual Support to Audio
+Adds a `gr.makeWaveform()` function that creates a waveform video by combining an audio and an optional background image by [@dawoodkhan82](http://github.com/dawoodkhan82) and [@aliabid94](http://github.com/aliabid94) in [PR 2706](https://github.com/gradio-app/gradio/pull/2706. Helpful for making audio outputs much more shareable.
```

Comment:
nit: Would be really cool to embed a gif of the waveform here. If you edit from the github ui directly, you can drag-and-drop a gif.

Reply:
done